generate_sl.py

```diff
@@ -2,13 +2,16 @@
 import argparse
 import os
 import glob
+import getpass
+import time
+from datetime import datetime
 import checkpoint
 from qcore import utils
 
 TEMPLATE_NAME = 'im_calc_sl.template'
 TIME = '00:30:00'
 DEFAULT_N_PROCESSES = 40
-
+DEFAULT_RRUP_OUTDIR = os.path.join('/home', getpass.getuser(),'imcalc_rrup_out_{}'.format(datetime.fromtimestamp(time.time()).strftime('%Y-%m-%d-%H-%M-%S')))
```

Comment:
Better to have a discussion about a better standard location for rrup calculations.